<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="1-2 Purchase Test Suite" time="26" tests="1" failures="1" errors="1">
   <testsuite name="1-2 Purchase Test Suite" tests="1" failures="1" errors="1" time="26" timestamp="2019-04-25 15:07:35" hostname="KennyIC_Chen - KennyIC_Chen.compal.com" id="Test Suites/1-2 Purchase Test Suite">
      <properties>
         <property name="deviceName" value=""/>
         <property name="devicePlatform" value=""/>
         <property name="logFolder" value="C:\\Users\\KennyIC_chen.COMPAL\\git\\PSItest\\Reports\\1-2 Purchase Test Suite\\20190425_150734"/>
         <property name="logFiles" value="C:\\Users\\KennyIC_chen.COMPAL\\git\\PSItest\\Reports\\1-2 Purchase Test Suite\\20190425_150734\\execution0.log"/>
         <property name="attachments" value="C:\\Users\\KennyIC_chen.COMPAL\\git\\PSItest\\Reports\\1-2 Purchase Test Suite\\20190425_150734\\1556176081067.png, C:\\Users\\KennyIC_chen.COMPAL\\git\\PSItest\\Reports\\1-2 Purchase Test Suite\\20190425_150734\\1556176081463.png"/>
         <property name="hostName" value="KennyIC_Chen - KennyIC_Chen.compal.com"/>
         <property name="os" value="Windows 10 64bit"/>
         <property name="katalonVersion" value="5.10.1.1"/>
         <property name="browser" value="Chrome 73.0.3683.103"/>
         <property name="hostAddress" value="10.109.102.148"/>
         <property name="sessionId" value="0721bf107a25c6d28b5ae676fdb45ac2"/>
         <property name="seleniumVersion" value="3.7.1"/>
         <property name="proxyInformation" value="ProxyInformation{proxyOption=NO_PROXY, proxyServerType=HTTP, password=, proxyServerAddress=, proxyServerPort=0}"/>
         <property name="platform" value="Windows 10"/>
      </properties>
      <testcase name="Test Cases/Others/Login" classname="Test Cases/Others/Login" status="PASSED">
         <system-out><![CDATA[2019-04-25 15:07:36 - [TEST_CASE][PASSED] - Test Cases/Others/Login: Test Cases/Others/Login

2019-04-25 15:07:36 - [TEST_STEP][PASSED] - openBrowser(""): Browser is opened with url: ''

2019-04-25 15:07:36 - [MESSAGE][INFO] - Starting 'Chrome' driver

2019-04-25 15:07:36 - [MESSAGE][INFO] - Action delay is set to 0 seconds

2019-04-25 15:07:39 - [MESSAGE][PASSED] - Browser is opened with url: ''

2019-04-25 15:07:39 - [TEST_STEP][PASSED] - navigateToUrl("https://psi-azcldap.unicore.com.tw/login"): Navigate to 'https://psi-azcldap.unicore.com.tw/login' successfully

2019-04-25 15:07:53 - [MESSAGE][PASSED] - Navigate to 'https://psi-azcldap.unicore.com.tw/login' successfully

2019-04-25 15:07:53 - [TEST_STEP][PASSED] - maximizeWindow(): Current window maximized

2019-04-25 15:07:54 - [MESSAGE][PASSED] - Current window maximized

2019-04-25 15:07:54 - [TEST_STEP][PASSED] - setText(findTestObject("Others/Login/ID-QWE"), "QWE"): Text 'QWE' is set on object 'Object Repository/Others/Login/ID-QWE'

2019-04-25 15:07:54 - [MESSAGE][PASSED] - Text 'QWE' is set on object 'Object Repository/Others/Login/ID-QWE'

2019-04-25 15:07:54 - [TEST_STEP][PASSED] - setEncryptedText(findTestObject("Others/Login/PW-QWE"), "5opu+sh6ATw="): Text ****** has been set on object 'Object Repository/Others/Login/PW-QWE'

2019-04-25 15:07:54 - [MESSAGE][PASSED] - Text ****** has been set on object 'Object Repository/Others/Login/PW-QWE'

2019-04-25 15:07:54 - [TEST_STEP][PASSED] - click(findTestObject("Others/Login/Login Button")): Object: 'Object Repository/Others/Login/Login Button' is clicked on

2019-04-25 15:07:54 - [MESSAGE][PASSED] - Object: 'Object Repository/Others/Login/Login Button' is clicked on

2019-04-25 15:07:54 - [TEST_STEP][PASSED] - verifyElementClickable(findTestObject("Others/Login/Verify button")): Object 'Object Repository/Others/Login/Verify button' is clickable

2019-04-25 15:07:55 - [MESSAGE][PASSED] - Object 'Object Repository/Others/Login/Verify button' is clickable]]></system-out>
         <system-err><![CDATA[]]></system-err>
      </testcase>
      <testcase name="Test Cases/1-2 Purchase-Add" classname="Test Cases/1-2 Purchase-Add" status="ERROR">
         <error type="ERROR" message="Test Cases/1-2 Purchase-Add FAILED.
Reason:
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
file:/C:/Users/KennyIC_chen.COMPAL/git/PSItest/Scripts/1-2%20Purchase-Add/Script1555921217692.groovy: 29: The current scope already contains a variable of the name day1
 @ line 29, column 9.
   	String day1 = day1.format('dd')
           ^

file:/C:/Users/KennyIC_chen.COMPAL/git/PSItest/Scripts/1-2%20Purchase-Add/Script1555921217692.groovy: 32: The current scope already contains a variable of the name day1
 @ line 32, column 9.
   	String day1 = day1.format('dd')
           ^

2 errors

	at com.kms.katalon.core.main.ScriptEngine.getScript(ScriptEngine.java:199)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:319)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:290)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:224)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:149)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1556176054272.run(TempTestSuite1556176054272.groovy:36)
"/>
         <system-out><![CDATA[2019-04-25 15:07:55 - [TEST_CASE][ERROR] - Test Cases/1-2 Purchase-Add: Test Cases/1-2 Purchase-Add FAILED.
Reason:
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
file:/C:/Users/KennyIC_chen.COMPAL/git/PSItest/Scripts/1-2%20Purchase-Add/Script1555921217692.groovy: 29: The current scope already contains a variable of the name day1
 @ line 29, column 9.
   	String day1 = day1.format('dd')
           ^

file:/C:/Users/KennyIC_chen.COMPAL/git/PSItest/Scripts/1-2%20Purchase-Add/Script1555921217692.groovy: 32: The current scope already contains a variable of the name day1
 @ line 32, column 9.
   	String day1 = day1.format('dd')
           ^

2 errors

	at com.kms.katalon.core.main.ScriptEngine.getScript(ScriptEngine.java:199)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:319)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:290)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:224)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:149)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1556176054272.run(TempTestSuite1556176054272.groovy:36)]]></system-out>
         <system-err><![CDATA[2019-04-25 15:07:55 - [TEST_CASE][ERROR] - Test Cases/1-2 Purchase-Add: Test Cases/1-2 Purchase-Add FAILED.
Reason:
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
file:/C:/Users/KennyIC_chen.COMPAL/git/PSItest/Scripts/1-2%20Purchase-Add/Script1555921217692.groovy: 29: The current scope already contains a variable of the name day1
 @ line 29, column 9.
   	String day1 = day1.format('dd')
           ^

file:/C:/Users/KennyIC_chen.COMPAL/git/PSItest/Scripts/1-2%20Purchase-Add/Script1555921217692.groovy: 32: The current scope already contains a variable of the name day1
 @ line 32, column 9.
   	String day1 = day1.format('dd')
           ^

2 errors

	at com.kms.katalon.core.main.ScriptEngine.getScript(ScriptEngine.java:199)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:319)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:290)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:224)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:149)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1556176054272.run(TempTestSuite1556176054272.groovy:36)]]></system-err>
      </testcase>
      <testcase name="Test Cases/Others/Accept and verify" classname="Test Cases/Others/Accept and verify" status="FAILED">
         <failure type="FAILED" message="Test Cases/Others/Accept and verify FAILED.
Reason:
com.kms.katalon.core.exception.StepFailedException: Unable to accept alert (Root cause: com.kms.katalon.core.exception.StepFailedException: No alert found)
	at com.kms.katalon.core.keyword.internal.KeywordMain.stepFailed(KeywordMain.groovy:36)
	at com.kms.katalon.core.webui.keyword.internal.WebUIKeywordMain.stepFailed(WebUIKeywordMain.groovy:65)
	at com.kms.katalon.core.webui.keyword.internal.WebUIKeywordMain.runKeyword(WebUIKeywordMain.groovy:27)
	at com.kms.katalon.core.webui.keyword.builtin.AcceptAlertKeyword.acceptAlert(AcceptAlertKeyword.groovy:71)
	at com.kms.katalon.core.webui.keyword.builtin.AcceptAlertKeyword.execute(AcceptAlertKeyword.groovy:66)
	at com.kms.katalon.core.keyword.internal.KeywordExecutor.executeKeywordForPlatform(KeywordExecutor.groovy:53)
	at com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords.acceptAlert(WebUiBuiltInKeywords.groovy:1492)
	at com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords$acceptAlert$7.call(Unknown Source)
	at Accept and verify.run(Accept and verify:18)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:319)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:290)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:224)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:149)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1556176054272.run(TempTestSuite1556176054272.groovy:36)
"/>
         <system-out><![CDATA[2019-04-25 15:07:55 - [TEST_CASE][FAILED] - Test Cases/Others/Accept and verify: Test Cases/Others/Accept and verify FAILED.
Reason:
com.kms.katalon.core.exception.StepFailedException: Unable to accept alert (Root cause: com.kms.katalon.core.exception.StepFailedException: No alert found)
	at com.kms.katalon.core.keyword.internal.KeywordMain.stepFailed(KeywordMain.groovy:36)
	at com.kms.katalon.core.webui.keyword.internal.WebUIKeywordMain.stepFailed(WebUIKeywordMain.groovy:65)
	at com.kms.katalon.core.webui.keyword.internal.WebUIKeywordMain.runKeyword(WebUIKeywordMain.groovy:27)
	at com.kms.katalon.core.webui.keyword.builtin.AcceptAlertKeyword.acceptAlert(AcceptAlertKeyword.groovy:71)
	at com.kms.katalon.core.webui.keyword.builtin.AcceptAlertKeyword.execute(AcceptAlertKeyword.groovy:66)
	at com.kms.katalon.core.keyword.internal.KeywordExecutor.executeKeywordForPlatform(KeywordExecutor.groovy:53)
	at com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords.acceptAlert(WebUiBuiltInKeywords.groovy:1492)
	at com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords$acceptAlert$7.call(Unknown Source)
	at Accept and verify.run(Accept and verify:18)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:319)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:290)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:224)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:149)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1556176054272.run(TempTestSuite1556176054272.groovy:36)


2019-04-25 15:07:55 - [TEST_STEP][PASSED] - waitForAlert(5): No alert found after 5 second(s).

2019-04-25 15:08:01 - [MESSAGE][WARNING] - No alert found after 5 second(s).

2019-04-25 15:08:01 - [TEST_STEP][FAILED] - acceptAlert(): acceptAlert() FAILED.
Reason:
com.kms.katalon.core.exception.StepFailedException: Unable to accept alert (Root cause: com.kms.katalon.core.exception.StepFailedException: No alert found)
	at com.kms.katalon.core.keyword.internal.KeywordMain.stepFailed(KeywordMain.groovy:36)
	at com.kms.katalon.core.webui.keyword.internal.WebUIKeywordMain.stepFailed(WebUIKeywordMain.groovy:65)
	at com.kms.katalon.core.webui.keyword.internal.WebUIKeywordMain.runKeyword(WebUIKeywordMain.groovy:27)
	at com.kms.katalon.core.webui.keyword.builtin.AcceptAlertKeyword.acceptAlert(AcceptAlertKeyword.groovy:71)
	at com.kms.katalon.core.webui.keyword.builtin.AcceptAlertKeyword.execute(AcceptAlertKeyword.groovy:66)
	at com.kms.katalon.core.keyword.internal.KeywordExecutor.executeKeywordForPlatform(KeywordExecutor.groovy:53)
	at com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords.acceptAlert(WebUiBuiltInKeywords.groovy:1492)
	at com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords$acceptAlert$7.call(Unknown Source)
	at Accept and verify.run(Accept and verify:18)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:319)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:290)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:224)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:149)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1556176054272.run(TempTestSuite1556176054272.groovy:36)


2019-04-25 15:08:01 - [MESSAGE][FAILED] - No alert found
[[ATTACHMENT|C:\\Users\\KennyIC_chen.COMPAL\\git\\PSItest\\Reports\\1-2 Purchase Test Suite\\20190425_150734\\1556176081067.png]]

2019-04-25 15:08:01 - [MESSAGE][FAILED] - Unable to accept alert (Root cause: com.kms.katalon.core.exception.StepFailedException: No alert found)
[[ATTACHMENT|C:\\Users\\KennyIC_chen.COMPAL\\git\\PSItest\\Reports\\1-2 Purchase Test Suite\\20190425_150734\\1556176081463.png]]

2019-04-25 15:08:01 - [MESSAGE][FAILED] - acceptAlert() FAILED.
Reason:
com.kms.katalon.core.exception.StepFailedException: Unable to accept alert (Root cause: com.kms.katalon.core.exception.StepFailedException: No alert found)
	at com.kms.katalon.core.keyword.internal.KeywordMain.stepFailed(KeywordMain.groovy:36)
	at com.kms.katalon.core.webui.keyword.internal.WebUIKeywordMain.stepFailed(WebUIKeywordMain.groovy:65)
	at com.kms.katalon.core.webui.keyword.internal.WebUIKeywordMain.runKeyword(WebUIKeywordMain.groovy:27)
	at com.kms.katalon.core.webui.keyword.builtin.AcceptAlertKeyword.acceptAlert(AcceptAlertKeyword.groovy:71)
	at com.kms.katalon.core.webui.keyword.builtin.AcceptAlertKeyword.execute(AcceptAlertKeyword.groovy:66)
	at com.kms.katalon.core.keyword.internal.KeywordExecutor.executeKeywordForPlatform(KeywordExecutor.groovy:53)
	at com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords.acceptAlert(WebUiBuiltInKeywords.groovy:1492)
	at com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords$acceptAlert$7.call(Unknown Source)
	at Accept and verify.run(Accept and verify:18)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:319)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:290)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:224)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:149)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1556176054272.run(TempTestSuite1556176054272.groovy:36)]]></system-out>
         <system-err><![CDATA[2019-04-25 15:07:55 - [TEST_CASE][FAILED] - Test Cases/Others/Accept and verify: Test Cases/Others/Accept and verify FAILED.
Reason:
com.kms.katalon.core.exception.StepFailedException: Unable to accept alert (Root cause: com.kms.katalon.core.exception.StepFailedException: No alert found)
	at com.kms.katalon.core.keyword.internal.KeywordMain.stepFailed(KeywordMain.groovy:36)
	at com.kms.katalon.core.webui.keyword.internal.WebUIKeywordMain.stepFailed(WebUIKeywordMain.groovy:65)
	at com.kms.katalon.core.webui.keyword.internal.WebUIKeywordMain.runKeyword(WebUIKeywordMain.groovy:27)
	at com.kms.katalon.core.webui.keyword.builtin.AcceptAlertKeyword.acceptAlert(AcceptAlertKeyword.groovy:71)
	at com.kms.katalon.core.webui.keyword.builtin.AcceptAlertKeyword.execute(AcceptAlertKeyword.groovy:66)
	at com.kms.katalon.core.keyword.internal.KeywordExecutor.executeKeywordForPlatform(KeywordExecutor.groovy:53)
	at com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords.acceptAlert(WebUiBuiltInKeywords.groovy:1492)
	at com.kms.katalon.core.webui.keyword.WebUiBuiltInKeywords$acceptAlert$7.call(Unknown Source)
	at Accept and verify.run(Accept and verify:18)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:319)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:290)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:224)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:149)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1556176054272.run(TempTestSuite1556176054272.groovy:36)]]></system-err>
      </testcase>
      <system-out><![CDATA[2019-04-25 15:07:35 - [TEST_SUITE][ERROR] - 1-2 Purchase Test Suite: Test Cases/1-2 Purchase-Add FAILED.
Reason:
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
file:/C:/Users/KennyIC_chen.COMPAL/git/PSItest/Scripts/1-2%20Purchase-Add/Script1555921217692.groovy: 29: The current scope already contains a variable of the name day1
 @ line 29, column 9.
   	String day1 = day1.format('dd')
           ^

file:/C:/Users/KennyIC_chen.COMPAL/git/PSItest/Scripts/1-2%20Purchase-Add/Script1555921217692.groovy: 32: The current scope already contains a variable of the name day1
 @ line 32, column 9.
   	String day1 = day1.format('dd')
           ^

2 errors

	at com.kms.katalon.core.main.ScriptEngine.getScript(ScriptEngine.java:199)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:319)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:290)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:224)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:149)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1556176054272.run(TempTestSuite1556176054272.groovy:36)]]></system-out>
      <system-err><![CDATA[2019-04-25 15:07:35 - [TEST_SUITE][ERROR] - 1-2 Purchase Test Suite: Test Cases/1-2 Purchase-Add FAILED.
Reason:
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
file:/C:/Users/KennyIC_chen.COMPAL/git/PSItest/Scripts/1-2%20Purchase-Add/Script1555921217692.groovy: 29: The current scope already contains a variable of the name day1
 @ line 29, column 9.
   	String day1 = day1.format('dd')
           ^

file:/C:/Users/KennyIC_chen.COMPAL/git/PSItest/Scripts/1-2%20Purchase-Add/Script1555921217692.groovy: 32: The current scope already contains a variable of the name day1
 @ line 32, column 9.
   	String day1 = day1.format('dd')
           ^

2 errors

	at com.kms.katalon.core.main.ScriptEngine.getScript(ScriptEngine.java:199)
	at com.kms.katalon.core.main.ScriptEngine.run(ScriptEngine.java:194)
	at com.kms.katalon.core.main.ScriptEngine.runScriptAsRawText(ScriptEngine.java:119)
	at com.kms.katalon.core.main.TestCaseExecutor.runScript(TestCaseExecutor.java:328)
	at com.kms.katalon.core.main.TestCaseExecutor.doExecute(TestCaseExecutor.java:319)
	at com.kms.katalon.core.main.TestCaseExecutor.processExecutionPhase(TestCaseExecutor.java:298)
	at com.kms.katalon.core.main.TestCaseExecutor.accessMainPhase(TestCaseExecutor.java:290)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:224)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestCaseMainPhase(TestSuiteExecutor.java:129)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:112)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:81)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:149)
	at com.kms.katalon.core.main.TestCaseMain$startTestSuite$0.call(Unknown Source)
	at TempTestSuite1556176054272.run(TempTestSuite1556176054272.groovy:36)]]></system-err>
   </testsuite>
</testsuites>
